ee/api/test/test_property_definition.py

```diff
@@ -64,6 +66,20 @@ def test_search_property_definition(self):
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         response_data = response.json()
         self.assertEqual(len(response_data["results"]), 1)
+        # always True if not scoping by event names
+        self.assertEqual(response_data["results"][0]["is_event_property"], True)
```

Comment:
See comment above on defaulting to `None`

Reply:
Sure, made it default to `null`